examples/06_benchmarks/movielens.ipynb

```diff
@@ -76,18 +76,18 @@
      "name": "stdout",
```

Comment:
<p>What GPU are you using?</p><p>I also noticed that you don't have CuDNN, I expected to get some speed up in the GPU algos if you install it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I'm using Nvidia Tesla P100.

For the CuDNN, I have it on my machine (version 7.6.5). I installed it using anaconda so it's not in one of the usual system paths, thus the `reco_utils.common.gpu_utils.get_cudnn_version` function couldn't detect it.